src/machines/java/com/enderio/machines/common/integrations/jei/MachinesJEI.java

```diff
@@ -54,4 +58,33 @@ public void registerRecipes(IRecipeRegistration registration) {
         registration.addRecipes(SoulBindingCategory.TYPE, recipes.getSoulBindingRecipes());
         registration.addRecipes(TankCategory.TYPE, recipes.getTankRecipes());
     }
+
+    @Override
+    public void registerRecipeTransferHandlers(IRecipeTransferRegistration registration) {
+        registration.addRecipeTransferHandler(AlloySmelterMenu.class, MachineMenus.ALLOY_SMELTER.get(), AlloySmeltingCategory.TYPE,
+            AlloySmelterMenu.INPUTS_INDEX, AlloySmelterMenu.INPUT_COUNT,
+            AlloySmelterMenu.LAST_INDEX + 1, 36);
+
+        registration.addRecipeTransferHandler(EnchanterMenu.class, MachineMenus.ENCHANTER.get(), EnchanterCategory.TYPE,
+            EnchanterMenu.INPUTS_INDEX, EnchanterMenu.INPUT_COUNT,
+            EnchanterMenu.LAST_INDEX + 1, 36);
+
+        registration.addRecipeTransferHandler(PrimitiveAlloySmelterMenu.class, MachineMenus.PRIMITIVE_ALLOY_SMELTER.get(), AlloySmeltingCategory.TYPE,
+            PrimitiveAlloySmelterMenu.INPUTS_INDEX, PrimitiveAlloySmelterMenu.INPUT_COUNT,
+            PrimitiveAlloySmelterMenu.LAST_INDEX + 1, 36);
+
+        registration.addRecipeTransferHandler(SagMillMenu.class, MachineMenus.SAG_MILL.get(), SagMillCategory.TYPE,
+            SagMillMenu.INPUTS_INDEX, SagMillMenu.INPUT_COUNT,
+            SagMillMenu.LAST_INDEX + 1, 36);
+
+        registration.addRecipeTransferHandler(SlicerMenu.class, MachineMenus.SLICE_N_SPLICE.get(), SlicingRecipeCategory.TYPE,
+            SlicerMenu.INPUTS_INDEX, SlicerMenu.INPUT_COUNT,
+            SlicerMenu.LAST_INDEX + 1, 36);
+
+        registration.addRecipeTransferHandler(SoulBinderMenu.class, MachineMenus.SOUL_BINDER.get(), SoulBindingCategory.TYPE,
```

Comment:
I'm not sure the soulbinder is in jei yet?

Reply:
It is, see line 58 of this file :)

Reply:
Oh right, but it doesn't show that you can rebind the broken spawner under uses. The soul binding recipe uses a mobspawner without a mob as an example.

Reply:
i swear it does 😅